Makefile

```diff
@@ -44,16 +67,4 @@ generate-release:
 	@./openshift/release/generate-release.sh v$(RELEASE_VERSION)
 .PHONY: generate-release
 
-push-image:
```

Comment:
Removing that for ?

Reply:
<img width="668" alt="image" src="https://user-images.githubusercontent.com/98980/61391120-5121d300-a8bc-11e9-90b2-61aa1b82cf36.png">